src/Pages/Home.js

```diff
@@ -2,12 +2,242 @@ import { NavLink } from "react-router-dom";
 import logo from "../images/logo.png";
 import { signOut } from "aws-amplify/auth";
 import { Button } from "@aws-amplify/ui-react";
-
+import { Authenticator, Text, useTheme, View, Image, Heading, useAuthenticator } from "@aws-amplify/ui-react";
 import TeamAccessor from "../Accessors/TeamAccessor";
 
+
+const components = {
```

Comment:
use arrow functions, and if you need to do something like this consider creating a new component in a separate file.

Reply:
check below